modules/log.sh

```diff
@@ -68,8 +68,9 @@ function log_error {
 }
 
 # Log the given message at FATAL level.
+FATAL_EXIT_CODE=21
```

Comment:
Why `21`? We should try to pick the most obscure number possible here to avoid it conflicting with actual exit codes.

Reply:
Yes, this is open for grabs obviously.

Which would you prefer?

I always pick my birthday if I have the opportunity, hence 21 :p

Reply:
https://tldp.org/LDP/abs/html/exitcodes.html#AEN23647
According to this, I think I'll go with `66` unless you object.

Reply:
`90`

http://www.tldp.org/LDP/abs/html/exitcodes.html

Reply:
haha, alright, `90` it is :p